packages/BetterPhpDocParser/Annotation/ChangeResolvedClassInParticularContextForAnnotation.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\BetterPhpDocParser\Annotation;
+
+use Rector\BetterPhpDocParser\PhpDoc\DoctrineAnnotationTagValueNode;
+use Rector\BetterPhpDocParser\ValueObject\PhpDoc\DoctrineAnnotation\CurlyListNode;
+use Rector\Php80\ValueObject\AnnotationToAttribute;
+
+class ChangeResolvedClassInParticularContextForAnnotation
+{
+    private const RULES = [
+        [
+            'tag' => 'Doctrine\ORM\Mapping\JoinTable',
+            'value' => 'inverseJoinColumns',
+            'resolved_class' => 'Doctrine\ORM\Mapping\InverseJoinColumns',
+        ],
+    ];
+
+    public function __invoke(
```

Comment:
This method name is a bit unclear. Could you update it to better decribe the operation?

Reply:
this class have only one responsability, the operation is describe by the class name ChangeResolvedClassInParticularContextForAnnotation. if you want i can name the method the same way

Reply:
i've name the method `changeResolvedClassIfNeed`